packages/wallets/src/utils/cfdUtils.ts

```diff
@@ -9,3 +9,8 @@ export const platformPasswordResetRedirectLink = (platform: TPlatforms.All, isVi
             return isVirtual ? 21 : 20;
     }
 };
+
+export const platformMapping: Record<string, Exclude<TPlatforms.All, 'ctrader'>> = {
+    trading_platform_dxtrade_password_reset: 'dxtrade',
```

Comment:
whats the platform mapping? what is mapped to what?

Reply:
So currently, there is no way for us to identify the `platform` based on the `action params` hence I have to result to this method to get the `platform`.

Reply:
okay, so this is the mapping of action to platform? 
e.g. to identify that specific action param triggers reset on specifc platform?